components/istio-kyma-patch/patch.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+
+set -e
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+
+if [[ -z ${CONFIG_DIR} ]]; then
+    CONFIG_DIR=${DIR}
+fi
+
+function run_all_patches() {
+  echo "--> Patch resources"
+  for f in $(find ${CONFIG_DIR} -name '*\.patch\.json' -maxdepth 1); do
+    local type=$(basename ${f} | cut -d. -f2)
+    local name=$(basename ${f} | cut -d. -f1)
+    echo "    Patch $type $name from: ${f}"
+    local patch=$(cat ${f})
+    set +e
+    local out
+    out=$(kubectl patch ${type} -n istio-system ${name} --patch "$patch" --type json)
+    set -e
+    local result=$?
```

Comment:
this `set -e` should be after `result` initialization otherwise `result` will be 0